src/components/Link.jsx

```diff
@@ -1,17 +1,3 @@
-import React from "react";
-import PropTypes from "prop-types";
-
-const Link = ({ children, className, ...props }) => {
-  return (
-    <a className={className} {...props}>
-      {children}
-    </a>
-  );
-};
-
-Link.propTypes = {
-  children: PropTypes.any,
-  className: PropTypes.string,
-};
+import { Link } from "react-router-dom";
```

Comment:
I like keeping this as a component, because we can add logic to make the handle base path wonkiness.